cmd/sharded-test-server/shard.go

```diff
@@ -98,6 +98,7 @@ func newShard(ctx context.Context, n int, args []string, standaloneVW bool, serv
 		fmt.Sprintf("--shard-client-cert-file=%s", shardClientCert),
 		fmt.Sprintf("--shard-client-key-file=%s", shardClientCertKey),
 		fmt.Sprintf("--shard-virtual-workspace-ca-file=%s", filepath.Join(workDirPath, ".kcp", "serving-ca.crt")),
+		"--service-account-lookup=false",
```

Comment:
TODO: make secure

cc @s-urbaniak @ncdc

Reply:
tl;dr:
Because the related secrets are not replicated, and the legacy lookup method code looks for the related secret in the current shard

Reply:
Where do we track the TODO ?

Reply:
Need an issue for this please, plus a TODO in the code I'd say